xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -461,6 +472,35 @@ namespace addon
      */
     virtual void PauseStream(double time) { }
 
+  /*!
+  * Return currently selected chapter
+  * @remarks
+  */
+  virtual int GetChapter() { return -1; };
+
+  /*!
+  * Return number of available chapters
+  * @remarks
+  */
+  virtual int GetChapterCount() { return 0; };
+
+  /*!
+  * Return name of chapter # ch
+  * @remarks
+  */
+  virtual const char* GetChapterName(int ch) { return nullptr; };
```

Comment:
The changes are looking good, only about here, is it possible to use `std::string`? Then it match more the C++ Interface.